contact.html

```diff
@@ -59,8 +59,13 @@ <h1 class="">ProgrammerGaurav</h1>
 								<input type="text" class="form-control" id="name" name="name" required>
 							</div>
 							<div class="form-group">
-								<label for="Messege">Messege</label>
-								<textarea type="email" class="form-control" id="messege" name="messege"
+								<label for="Email">Email</label>
```

Comment:
I redirected the user to my WhatsApp profile so no need to take email, I am not using any backend in this portfolio,